### ISSUE TITLE
Transaction download is failing for a Complete Access account

### ISSUE THREAD

**Issue by grassdog** [twang2218/node-cba-netbank#10]
Downloading seems to have stopped working for one of my accounts in the last week or so.

When I try to download from a Complete Access account using the debug build I see the following:

```
Downloading history [15/11/2017 => 15/02/2018] ...
  node-cba-netbank getTransactionHistory(account: Complete Access [XXXXX] => 90.92) +0ms
  node-cba-netbank GET https://www2.my.commbank.com.au/netbank/TransactionHistory/History.aspx?ACCOUNT_... +1ms
  node-cba-netbank headers => {} +0ms
  node-cba-netbank parseTitle(): found title => 'Transactions' +1s
Unhandled rejection Cannot find transactions in the resp
```

Other accounts still seem to work fine.

I'll try to debug what's going on but I'm guessing you'd be able to see what's happening more quickly 😄

**Comment by grassdog** [comment]
I've found the problem. The regex at https://github.com/twang2218/node-cba-netbank/blob/master/src/parser.js#L231 chokes if there are any embedded `;` in the transaction data. So if you have a transaction with the following text `"Text":"IVY &amp; LARK"` the regex doesn't match.

I'll put up a PR to make the regex a little more permissive.

**Comment by twang2218** [comment]
They used to return `&` in JSON, then encoded by the JS before display on the page, maybe they changed their mind and decided to encode the string in JSON, so the message can be displayed directly. I think the previous way is the right way to do, data is just data.

I saw your PR, LGTM, so I merged it. Thank you very much.

**Comment by grassdog** [comment]
Good stuff. Thanks for merging!

**Comment by grassdog** [comment]
Any chance you can publish a new version to npm with this change included?

**Comment by twang2218** [comment]
Just published as version `0.8.2`, it seems to have a problem with Node 4 as I updated the dependencies as well, but I think it should be fine, as not many people still using Node 4, will drop the Node 4 support anyway.

**Comment by grassdog** [comment]
Thanks!
